tools/pipelines/build-client.yml

```diff
@@ -22,6 +22,11 @@ parameters:
     - default
     - skip
     - force
+- name: nonScopedPackages
```

Comment:
I wonder why you make this into a parameter vs a variable. Do you intent to be overridable by manually queueing a job?

Reply:
Yes, I was thinking that manual queues created could update the list if they want to test adding another unscoped package. The default value will always be there so that the build runs successfully even when the build is not manually run